python/cugraph-pyg/cugraph_pyg/tests/mg/test_mg_cugraph_store.py

```diff
@@ -385,7 +385,7 @@ def test_get_input_nodes(karate_gnn, dask_client):
 def test_mg_frame_handle(graph, dask_client):
     F, G, N = graph
     cugraph_store = CuGraphStore(F, G, N, multi_gpu=True)
-    assert isinstance(cugraph_store._EXPERIMENTAL__CuGraphStore__graph._plc_graph, dict)
+    assert isinstance(cugraph_store.__graph._plc_graph, dict)
```

Comment:
Shouldn't this be `_CuGraphStore__graph`?

Reply:
I am not very familiar with the class, but I did not see `_CuGraphStore__graph` attribute under `cugraph_pyg.data.CuGraphStore`.

Reply:
It's a private variable, you need the `_CuGraphStore` prefix before the variable name

Reply:
`_plc_graph` is not private so it doesn't require that prefix

Reply:
but `__graph` is since it begins with two underscores

Reply:
Yeah, I see what you meant, will fix all usages.